extensions/shelly/actions/generateMessage/generateMessage.ts

```diff
@@ -0,0 +1,56 @@
+import { Category, type Action } from '@awell-health/extensions-core'
+import { generateMessageWithLLM } from './lib/generateMessageWithLLM'
+import { validatePayloadAndCreateSdk } from '../../lib'
+import { type settings } from '../../settings'
+import { fields, dataPoints, FieldsValidationSchema } from './config'
+
+export const generateMessage: Action<
+  typeof fields,
+  typeof settings,
+  keyof typeof dataPoints
+> = {
+  key: 'generateMessage',
+  category: Category.WORKFLOW,
+  title: 'Generate Message',
+  description:
+    'Generate a personalized message',
+  fields,
+  previewable: false,
+  dataPoints,
+  onEvent: async ({ payload, onComplete, onError, helpers }): Promise<void> => {
+    const {
+      ChatModelGPT4o,
+      fields: { communicationObjective, personalizationInput, additionalInstructions, stakeholder, language },
+    } = await validatePayloadAndCreateSdk({
+      fieldsSchema: FieldsValidationSchema,
+      payload,
+    })
+
+    try {
+      const generated_message = await generateMessageWithLLM({
+        ChatModelGPT4o,
+        communicationObjective,
+        personalizationInput,
+        additionalInstructions,
+        stakeholder,
+        language,
+      })
+      console.log('Hereeee')
```

Comment:
Do we want to keep the console logs?

Reply:
This one no 😅 , I will leave the rest for now.